unturned/OpenMod.Unturned/Vehicles/Events/UnturnedVehicleStealBatteryEvent.cs

```diff
@@ -0,0 +1,22 @@
+﻿using OpenMod.API.Eventing;
+using OpenMod.Unturned.Players;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenMod.Unturned.Vehicles.Events
+{
+    public class UnturnedVehicleStealBatteryEvent : UnturnedVehicleEvent, ICancellableEvent
```

Comment:
Rename to `UnturnedVehicleStealingBatteryEvent` for consistency
Prior to action - present participle (usually ending in 'ing')
After action - past tense (usually ending in 'ed')